src/Component/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import { AvatarCompSize, AvatarImgSize } from "../../variables";
+
+export interface ImgProps {
+    imgsrc?: string;
+    order: number
+};
+
+const ImgComp = styled.div<ImgProps>`
+height:${AvatarCompSize.max_height};
+width:${AvatarCompSize.max_width};
+border-radius:50%;
+display:flex;
+align-item:center;
+order:${props=>props.order};
+`;
+
+
+export const Avatar: React.FC<ImgProps> = (
+    props: ImgProps
+) => {
+    const { imgsrc,order } = props;
+    return (
+        <ImgComp order={order}><img style={{ height: AvatarImgSize.height, width: AvatarImgSize.width, display: "block", margin: "auto" }} src={imgsrc} alt="bot" /></ImgComp>
```

Comment:
You can add relative height and width in percentage here as it is inside ImgComp and can make styled component for img tag

```diff
- <img style={{ height: AvatarImgSize.height, width: AvatarImgSize.width, display: "block", margin: "auto" }} src={imgsrc} alt="bot" />
+ <img style={{ height: "100%", width: "100%", display: "block", margin: "auto" }} src={imgsrc} alt="bot" />
```